inventory_layouts/duffy_3node_inventory_layout.yml

```diff
@@ -0,0 +1,41 @@
+---
+inventory_layout:
+  vars:
+    openshift_hostname: __IP__
+    openshift_public_hostname: __IP__
+  hosts:
+    openshift-master:
+      count: 1
+      host_groups:
+        - masters
+        - nodes
+        - OSEv3
+    openshift-node:
+      count: 1
+      host_groups:
+        - nodes
+        - OSEv3
+    openshift-repo-host:
+      count: 1
+      host_groups:
+        - nodes
+        - OSEv3
+        - repo_host
+  host_groups:
+    OSEv3:
+      vars:
+        openshift_docker_additional_registries: |
+            "registry.example.com"
+        openshift_docker_insecure_registries: |
+            "registry.example.com"
+        debug_level: 2
+        ansible_sudo: False
+        ansible_ssh_user: root
+        openshift_override_hostname_check: True
+        openshift_set_hostname: True
+      children:
+        - masters
+        - nodes
+    masters: ""
+    nodes: ""
+    repo_host: "" 
```

Comment:
This file is poorly named. I realized this recently. It should be named openshift-3node-cluster.yml or something similar. The goal of this layout is not to spin up duffy instances, but to generate inventory file(s) that can be used to create/run an openshift cluster.